HW_3/templates/registration.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'base.html' %}
+{% block title %}
+Registration 3
+{% endblock %}
+
+{% block content %}
+    <h1>Registration 3 page</h1>
+    <form method="POST" action="{{ url_for('registration3') }}">
+        {{ form.csrf_token }}
+        {% with messages = get_flashed_messages(with_categories=true) %}
+            {% if messages %}
+                {% for category, message in messages %}
+                    <div class="alert alert-{{ category }}">
+                        {{ message }}
+                    </div>
+                {% endfor %}
+            {% endif %}
+        {% endwith %}
+        <p>
+            {{ form.name.label }}<br>
+            {{ form.name(size=32) }}
+        </p>
+        <p>
+            {{ form.surname.label }}<br>
+            {{ form.surname(size=32) }}
+        </p>
+        <p>
+            {{ form.email.label }}<br>
+            {{ form.email(size=32) }}
+        </p>
+        <p>
+            {{ form.password.label }}<br>
+            {{ form.password(size=32) }}
+        </p>
+        <p>
+            {{ form.confirm_password.label }}<br>
+            {{ form.confirm_password(size=32) }}
+        </p>
+        <p>
+            {{ form.check.label }}<br>
+            {{ form.check }}
+        </p>
+        <p>
+        <input type="submit" value="Login">
+        </p>
```

Comment:
Тут можно иначе. Используйте циклы
```suggestion
            {% for field in form if obj.name != "csrf_token" %}
	        <p> {{ field.label }}
	                {{ field(size=32) }}
	        </p>
	                {% for error in field.errors %}
	                    <span style="color: red;">{{ error }}</span><br>
	                {% endfor %}
	    {% endfor %}
```
Причем строки
```html
	                {% for error in field.errors %}
	                    <span style="color: red;">{{ error }}</span><br>
	                {% endfor %}
```
будут повторять flash сообщение об ошибке. Тут вы можете решить, что лучше: либо выводить сообщение тут, либо выше.